docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,132 @@
+# Release 381 (13 May 2022)
+
+## General
+
+* Add support for table functions. ({issue}`1839`, {issue}`11937`)
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage . ({issue}`12211`)
+* Increase the default number of stages allowed for a query from 100 to 150,
+  specified with `query.max-stage-count`, so large queries do not fail. ({issue}`12292`)
+* Use the `fault_tolerant_execution_partition_count` session property, or the
+  `fault-tolerant-execution-partition-count` configuration property, to adjust
+  the number of partitions for distributed joins and aggregations executed with
+  fault-tolerant execution enabled (`retry-policy=TASK`). ({issue}`12263`)
+
+## Security
+
+* Add support for automatic discovery of OpenID Connect metadata with OAuth 2.0
+  authentication. ({issue}`9788`)
+* Re-introduce `ldap.ssl-trust-certificate` as legacy configuration to avoid
+  update failures. ({issue}`12187`)
+* Fix query failure when a table has multiple column masks defined. ({issue}`12262`)
+* Fix regression in file-based system and connector access controls. ({issue}`12204`)
+* Fix regression in LDAP password authenticator usage with ActiveDirectory. ({issue}`12231`)
```

Comment:
@Praveen2112  .. is this okay?

Reply:
Same here.

Reply:
thats actually PR 12310 .. and if fixes the isseue 12321

I just removed "regression in" because from the PR I think this is a new fix .. please confirm @Praveen2112